src/components/HomePage/HomePage.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import { Button, InputText } from './../../utils';
+import { Movie } from './Movie/Movie';
+import { Filters } from './Filters/Filters';
+import { Footer } from './../Footer/Footer';
+import { Icon } from './../Icon/Icon';
+import { films } from '../../MockedData';
+import styles from './HomePage.css';
+
+console.log('styles', styles);
+
+const HomePage = () => {
+  const [moviesCount, setMoviesCount] = useState(39);
```

Comment:
you need to create a constant for magic numbers

Reply:
I changed it to initial value https://github.com/Azaliyaa/netflix-roulette/pull/1/commits/c9362e737f87aed20a979cc23bf00d54e63dd86d